src/activate_linux.c

```diff
@@ -181,6 +201,8 @@ int main(int argc, char *argv[]) {
                 HELP("-s scale\tScale ratio (float)");
                 HELP("-H offset\tMove overlay horizontally (integer)");
                 HELP("-V offset\tMove overlay  vertically  (integer)");
+                HELP("-S screen_list\tSelect screens to display the message");
```

Comment:
Please, add example of using this argument into [ARGS.md](https://github.com/MrGlockenspiel/activate-linux/blob/main/ARGS.md) file

Reply:
Yep